spec/system/login_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+RSpec.describe 'Logins', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+  describe 'Login page' do
+    before :each do
+      visit user_session_path
+    end
+    it 'should have login text' do
+      expect(page).to have_content 'Log in'
+    end
+    it 'should have input email' do
+      expect(page).to have_content 'Email'
+    end
+    it 'should have password input' do
+      expect(page).to have_content 'Password'
+    end
+    it 'should have login button' do
```

Comment:
- Kindly make sure the login test is passing
![Screenshot from 2022-07-29 20-40-58](https://user-images.githubusercontent.com/44624138/181833302-31011cfc-550b-4b96-b5c0-fef384f036ff.png)

Reply:
Thanks for your kind review, we made some changes after the test and we forgot to check the test and run the tests again, and after requesting for Review we noticed that there was one test failing, although we did fix it I guess we were too late.